tests/test_operations.py

```diff
@@ -550,3 +545,49 @@ def test_operation_without_handler(self):
         future2 = set_service.set_string(operation_handle=operation_handle, requested_string=value)
         result2 = future2.result(timeout=SET_TIMEOUT)
         self.assertGreater(result2.InvocationInfo.TransactionId, result.InvocationInfo.TransactionId)
+
+    def test_delayed_processing(self):
+        """Verify that flag 'delayed_processing' changes responses as expected."""
+        logging.getLogger('sdc.client.op_mgr').setLevel(logging.DEBUG)
+        logging.getLogger('sdc.device.op_reg').setLevel(logging.DEBUG)
+        logging.getLogger('sdc.device.SetService').setLevel(logging.DEBUG)
+        logging.getLogger('sdc.device.subscrMgr').setLevel(logging.DEBUG)
+        set_service = self.sdc_client.client('Set')
+        client_mdib = ConsumerMdib(self.sdc_client)
+        client_mdib.init_mdib()
+        coding = pm_types.Coding(NomenclatureCodes.MDC_OP_SET_TIME_SYNC_REF_SRC)
+        my_operation_descriptor = self.sdc_device.mdib.descriptions.coding.get_one(coding, allow_none=True)
+
+        operation_handle = my_operation_descriptor.Handle
+        operation = self.sdc_device.get_operation_by_handle(operation_handle)
+        for value in ('169.254.0.199', '169.254.0.199:1234'):

```

Comment:
why these ips?

Reply:
why not? Test needssome different ip addresses, exact value does not matter.